src/transport/mqtt/config/tls.rs

```diff
@@ -131,13 +131,16 @@ async fn read_root_cert_store() -> Result<RootCertStore, PairingError> {
     tokio::task::spawn_blocking(|| {
         let mut root_cert_store = RootCertStore::empty();
 
-        let native_certs =
-            rustls_native_certs::load_native_certs().map_err(PairingError::Native)?;
+        let res = rustls_native_certs::load_native_certs();
 
-        for cert in native_certs {
-            root_cert_store.add(cert).map_err(PairingError::Tls)?;
+        for err in res.errors {
+            error!(error = %crate::error::Report::new(err), "couldn't load root certificate");
         }
 
+        let (added, ignored) = root_cert_store.add_parsable_certificates(res.certs);
+
+        debug!("loaded {added} certs and {ignored} ignored");
```

Comment:
Do we want to add these logs?
```suggestion
```

Reply:
We could make it a trace, but it should only be printed once at startup